lib/HttpClient.ts

```diff
@@ -88,7 +88,8 @@ export class HttpClient {
     private _httpProxyBypassHosts: RegExp[];
     private _allowRedirects: boolean = true;
     private _maxRedirects: number = 50
-
+    private _agent;
+    private _usePrivateAgent;
```

Comment:
this is no longer "private" agent.  it's whether we reuse connections.  rename

Reply:
done